features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/conf/locales/jaggery/locale_default.json

```diff
@@ -277,7 +277,7 @@
     "useCase1":"WSO2 Use Case: API Facade Pattern",
     "more":"More >>",
     "throttlingTier":"Throttling Tier",
-	"sequence":"Sequences",
+	"sequence":"Message Mediation Policies",
```

Comment:
Fixed in #1494 (https://github.com/wso2/carbon-apimgt/commit/26d83cadacefd78bd06de37ab85d91adc40e8945)